### ISSUE TITLE
continued library issues

### ISSUE THREAD

**Issue by oldgrizz** [duinoapp/duinoapp-client#25]
cant access libraries or manually add libraries.  example, LiquidCrystal.  I am trying to use chromebooks to program arduino clones outside the arduino ide in an educational setting (jr/high school).  I need to be able to utilize libraries in order to explore other inputs and outputs.  Any help would be great.

**Comment by mrfrase3** [comment]
Hey, can you please describe the steps your taking to use the library?

**Comment by tjbocch813** [comment]
Okay, so I have run into a couple problems. My biggest problem is getting the program to compile/upload if I am using MULTIPLE libraries.

I have been using the Servo and NewPing libraries and they work fine when I am using just one, but add them both in there and I cannot get anything to compile.

Also make sure that you are only using one library for ALL of the files in your project's folder. Even using different ones in different files have caused problems for me.

This doesn't solve much, but hopefully helps understand the problem.

**Comment by mrfrase3** [comment]
That's certainly interesting, I'll have to investigate further. I'll post in here any updates.

**Comment by stephencunningham4** [comment]
I can confirm I am also having this exact issue.

**Comment by ran-to** [comment]
I can also confirm.

Simplest test to do to replicate this issue:
* add library "Servo", version 1.1.8
* Run the "Check & Compile"
Result: Everything compiles

* add also library "ezTime", or any other library of free choise
* Run the "Check & Compile"
Result: "**0% - Initialising Libraries...**" - and stuck there

It does not matter which order the libraries have been added. the combo of "Servo" library together with others gets stuck.
It is not a problem of number of libraries added either, as adding several other libraries together works fine.

To rule out other issues with libraries, I launched the Arduino Cloud platform instead to test.
There everything compiled with same libraries added.

I also installed my own Duino compiling server with docker on windows machine to test.
Same issue with duino app with that server on localhost compiling.  

Let me know if you would like me to test something more, to figure the bug out on the duino app/server.

Newbie on Arduino stuff and wanted to stay only on my chromebook.
Got really scared getting so much issues early on.
I am not sure how to proceed, but most likely I'll do my coding without this library to the extent I can and then copy code over, add libraries and do final compiling on another cloud platform.
